public/manifest.json

```diff
@@ -20,6 +21,8 @@
   ],
   "start_url": ".",
   "display": "standalone",
+  "orientation": "portrait",
   "theme_color": "#000000",
```

Comment:
I think we should change the theme color and maybe the background color to something from our palette

Reply:
Good idea. Updating that now!